images/build/go-runner/cloudbuild.yaml

```diff
@@ -9,6 +9,12 @@ options:
   machineType: 'N1_HIGHCPU_8'
 
 steps:
+  - name: 'gcr.io/projectsigstore/cosign/ci/cosign@sha256:c581a4f0f6dd158220fa05d2351d8015f969b68de5c61d52c41478fae84e7064' # v0.3.1
+    args:
+    - 'verify'
+    - '-key'
+    - 'https://raw.githubusercontent.com/GoogleContainerTools/distroless/main/cosign.pub'
```

Comment:
Do we wanna pin this to an actual version/sha? If the `main` branch changes then the build will silently fail.

Reply:
@priyawadhwa set this up intentionally - but she's out this week so I'll try to document the rationale in that repo.

The key rotation strategy for distroless is that we'll commit a new public key into the repo, then the images are built and signed with the corresponding private key. Since K8s takes distroless only from HEAD, the only appropriate key to check against is latest.

The branch name one is a good point though - is there a good way to access whatever the default branch is?

Reply:
done :)